### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by 0bs72** [pxsocs/warden_terminal#94]
I'm getting this warning :

```
/home/admin/.local/lib/python3.8/site-packages/apscheduler/triggers/interval.py
    :66: PytzUsageWarning:
        The normalize method in no longer necessary, as this time zone supports the fold
        attribute (PEP 495). For more details on migrating to a PEP 495-compliant
        implementation, see
        https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html 
    return self.timezone.normalize(next_fire_time)
```

I will look into how to fix this and submit a PR if I find a fix, but my skills are not on par with yours.

I like this, because I can simply ssh into my Nodebox (running on a VM, hosted on my PowerEge-T410) and fire up WARden.

Thanks for your hard work.

Peace.

**Comment by tehelsper** [comment]
@pxsocs I'm trying to add this to myNode and I'm seeing this info as well. I get scrolling errors that prevent the output from being visible.

**Comment by pxsocs** [comment]
Hi - been trying to find the source of this issue. 
Where does the message above show? Does it run at all? 
Anyway you can send a screenshot?

**Comment by pxsocs** [comment]
Actually found a potential issue pointed [here](https://stackoverflow.com/questions/69776414/pytzusagewarning-the-zone-attribute-is-specific-to-pytzs-interface-please-mig). 

Deployed a fix on version 0.707. Can you test and let me know if resolved?

**Comment by 0bs72** [comment]
> Actually found a potential issue pointed [here](https://stackoverflow.com/questions/69776414/pytzusagewarning-the-zone-attribute-is-specific-to-pytzs-interface-please-mig).
> 
> Deployed a fix on version 0.707. Can you test and let me know if resolved?

Yeah, V: 0.0.707 seems to have fixed the error that I reported on.
Now that I've a steady display (my thanks to you for that) I can do some more testing and see if any other issues arise; that is of course if you want such reports -- I take nothing for granted.

As I said before, I'm running this on my server and as such, I don't have (or have the need for) a large screen attached. In fact I run it 'headless' for the most part, but I have the option to connect it to a very large LCD if needs be. So I'll do that over the next few days (time permitting) and see how it looks.

You've obviously invested a lot of time in this; nice job mate.

With thanks.
